recipes/filter-api-validator.rb

```diff
@@ -0,0 +1,35 @@
+#
+# Cookbook Name:: repose
+# Recipe:: filter-header-normalization
+#
+# Copyright (C) 2013 Rackspace Hosting
+#
+
+include_recipe 'repose::install'
+
+unless node['repose']['filters'].include? 'api-validator'
+  filters = node['repose']['filters'] + ['api-validator']
+  node.normal['repose']['filters'] = filters
+end
+
+template "#{node['repose']['config_directory']}/validator.cfg.xml" do
+  owner node['repose']['owner']
+  group node['repose']['group']
+  mode '0644'
+  variables(
+    :multi_role_match => node['repose']['api_validator']['multi_role_match'],
+    :delegating_quality => node['repose']['api_validator']['delegating_quality'],
+    :validators => node['repose']['api_validator']['validators']
+  )
+  notifies :restart, 'service[repose-valve]'
+end
+
+cookbook_file "allfeeds_observer.wadl" do
```

Comment:
Made the change in latest commit.  Since a wadl is associated with each validator element, the recipe iterates through each validator in the configuration attributes and renders the file name from there.